### PR TITLE
Create README-Dutch.md

### LINKED ISSUE #1355: [Translate] README Dutch translation
### Summary

As requested in #1335, I'm starting with translating the README to Dutch

### What needs to be done?

- [ ] Translate the README.md in Dutch

### Who can help?

* *Anyone with some Dutch knowledge :)

### PR DESCRIPTION
**Pull request by EstherPlomp**
### Summary

Fixes #1355 - part of #1335 

### List of changes proposed in this PR (pull-request)

* *Add a Dutch translation of the README.md file (README-Dutch.md)

### What should a reviewer concentrate their feedback on?

- [ ] Klopt mijn Nederlands? :) 
- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

**Review by StefanJanssen95 — COMMENTED** [review]
The sections that have been translated seem correct, but a big part still isn't translated. I also noticed the file name is missing the ".md" extension so Github doesn't know it needs to be rendered as a markdown file.

**Comment by EstherPlomp** [comment]
Done with the translation! If anyone could please check whether I missed something and whether my Dutch is sufficient enough, it would be great :)

File: README-Dutch [hunk 1]
```diff
@@ -39,48 +39,50 @@ De introductie van _The Turing Way_ begint vanaf 5:13.
...
+Reproduceerbaar onderzoek is nodig om ervoor te zorgen dat wetenschappelijk werk kan worden vertrouwd.
+Financiers en uitgevers beginnen te eisen dat publicaties de onderliggende data en de analysecode bevatten.
+Het doel van reproduceerbaar onderzoek is om ervoor te zorgen dat alle resultaten onafhankelijk kunnen worden geverifieerd en gebruikt kunnen worden in toekomstig onderzoek.
+Dit is soms makkelijker gezegd dan gedaan.
+Het delen van deze onderzoeksresultaten betekent inzicht in gegevensbeheer, bibliotheekwetenschappen, softwareontwikkeling en continue integratietechnieken: vaardigheden die niet algemeen worden onderwezen of verwacht van academische onderzoekers en datawetenschappers. Aangezien deze activiteiten niet algemeen worden onderwezen, erkennen we dat de last van vereisten en het verwerven van nieuwe vaardigheden intimiderend kan zijn voor personen die nieuw zijn in deze wereld.
+* The Turing Way * is een handboek om studenten, hun supervisors, financiers en tijdschriftenredacteuren te ondersteunen om ervoor te zorgen dat reproduceerbare datawetenschap "te gemakkelijk is om niet te doen", zelfs voor mensen die nog nooit op deze manier hebben gewerkt.
```
Comment (StefanJanssen95) [inline README-Dutch:47]: The term "supervisors" is used in Dutch day to day language, but isn't a Dutch word. I'm not sure if we have a word for this in Dutch, but somethings that come close are "opzichter" (not really used a lot) or in this context maybe "begeleider"?

**Review by StefanJanssen95 — CHANGES_REQUESTED** [review]
There are minor things that should be improved but overall the text is correctly translated.
The file still requires a rename to "README-Dutch.md" to let Github format the file correctly.

File: README-Dutch [hunk 2]
```diff
@@ -39,48 +39,50 @@ De introductie van _The Turing Way_ begint vanaf 5:13.
...
 ### Bijdragen
 
-:construction: This repository is always a work in progress and **everyone** is encouraged to help us build something that is useful to the many. :construction:
+:construction: Aan deze repository wordt altijd gewerkt en ** iedereen ** wordt aangemoedigd om ons te helpen iets te bouwen dat nuttig is voor velen. :construction:
 
-Everyone is asked to follow our [code of conduct](CODE_OF_CONDUCT.md) and to checkout our [contributing guidelines](CONTRIBUTING.md) for more information on how to get started.
+Wij vragen iedereen zich te houden aan de [code of conduct](CODE_OF_CONDUCT.md) en onze [contributing guidelines](CONTRIBUTING.md) te lezen voor meer informatie over hoe je kan beginnen met bijdragen. 
 
-If you are not familiar or confident contributing on GitHub, you can also contribute a case study and your tips and tricks via our [Google submission form](https://goo.gl/forms/akFqZEIy2kxAjfZW2).
+Als je niet bekend of zelfverzekerd bent met GitHUB kan je ook een case study of je tips en trucs bijdragen via een [Google submission form](https://goo.gl/forms/akFqZEIy2kxAjfZW2).
```
[the remark this answers is not in the document]
Comment (EstherPlomp) [reply README-Dutch:64]: Thanks!

File: README-Dutch [hunk 3]
```diff
@@ -39,48 +39,50 @@ De introductie van _The Turing Way_ begint vanaf 5:13.
...
-Voor meer informatie over hoe je ons kan contacteren, ga naar [ways of working document](ways_of_working.md).
+Dit is (een deel) van het project team wat aan het werk is op het Turing Institute.
+Voor meer informatie over hoe je ons kan contacteren zie [ways of working document](ways_of_working.md).
 
 ![Team photo](book/website/figures/TuringWayTeam.jpg)
 
 ### Bijdragen
 
-:construction: This repository is always a work in progress and **everyone** is encouraged to help us build something that is useful to the many. :construction:
+:construction: Aan deze repository wordt altijd gewerkt en ** iedereen ** wordt aangemoedigd om ons te helpen iets te bouwen dat nuttig is voor velen. :construction:
```
[the remark this answers is not in the document]
Comment (EstherPlomp) [reply README-Dutch:60]: I don't think it should matter that much but it is indeed nicer to remove the spaces, thanks!

File: README-Dutch (hunk 1 above)
Comment (EstherPlomp) [reply README-Dutch:47]: Thanks! I went with begeleider as that is a good Dutch substitute indeed :)

**Comment by EstherPlomp** [comment]
I updated the file name (whoops) and implemented all the suggested changes. Please let me know if there is anything else!

**Review by StefanJanssen95 — APPROVED** [review]
Everything seems fine to me 👍

**Comment by EstherPlomp** [comment]
@BrainonSilicon I think we're good to go with this one!

**Comment by malvikasharan** [comment]
Thanks @EstherPlomp and @StefanJanssen95 for the fantastic work. We absolutely could not have done this on our own (literally no Dutch knowledge here!). I am approving this and leaving for you to merge.

**Review by malvikasharan — APPROVED** [review]

**Pull request merged**